lib/utils/background_push.dart

```diff
@@ -403,3 +403,26 @@ class BackgroundPush {
     );
   }
 }
+
+class UPFunctions extends UnifiedPushFunctions {
```

Comment:
Hm, this looks somehow generic. Isn't there a default like this in the unifiedpush_ui package itself we could use instead?

Reply:
I agree with you but that's the way [up docs](https://unifiedpush.org/developers/flutter/#register-for-pusl) show it. According to [API docs](https://pub.dev/documentation/unifiedpush_ui/latest/unifiedpush_ui/UnifiedPushFunctions-class.html) the methods in question are abstract. So it seems like we need our own implementation :/